subgraphs/arrakis-finance/schema.graphql

```diff
@@ -486,6 +505,20 @@ type VaultDailySnapshot @entity {
 
   " Timestamp of this snapshot "
   timestamp: BigInt!
+
+  # TODO: for diagnosing, to be removed
```

Comment:
another good catch! After working on the arrakis subgraph a bit more, I have a change of mind. I think it makes sense to keep _token0Amount and _token1Amount etc as optional fields - they may make sense to add to the standard schema - because it is hard to get those information from the current schema. let me know what you think.

I removed the TODO comments.